components/tabs/metadata/tabs.yml

```
@@ -448,3 +448,22 @@ examples:
       </div>
 
       <div class="dummy-spacing"></div>
+  - id: tabs
+    name: Tabs with anchors
+    status: Verified
+    markup: |
+      <div class="spectrum-Tabs spectrum-Tabs--horizontal">
+        <div class="spectrum-Tabs-item is-selected" tabindex="0">
+          <a href="#1" class="spectrum-Tabs-itemLabel">Tab 1</a>
+        </div>
```

```suggestion
        <a href="#1" class="spectrum-Tabs-item is-selected" tabindex="0">
          <span class="spectrum-Tabs-itemLabel">Tab 1</span>
        </a>
```

This creates a double tab stop. Maybe we want to update the `item` to an `<a>` it has all of the hover/focus/etc styles anyways?

---

Or maybe the right answer is just to remove the manual application of `tabindex` in this example markup? Hard to say.

---

Good call, it sounds like we should have `<a class="spectrum-Tabs-item">` and retain the existing `<span>` label.